website/blog/2023-10-09-dso-toolkit-next.mdx

```diff
@@ -0,0 +1,60 @@
+---
+authors: [anneke]
+---
+
+# DSO Toolkit Next
+
+We hebben een nieuw component erbij: het logo! Je kunt nu door middel van dit component gemakkelijk een smal logo in de header plaatsen. Om te laten zien hoe dat werkt hebben we het header component in Storybook ook aangepast.
+
+## Logo
+
+In de oude situatie gaf je één of twee statische afbeeldingen door aan het header component. Een afbeelding voor het logo (een afbeelding van het beeldmerk en het woordmerk) en een afbeelding met daarin de applicatienaam.
+
+In het component laden we nu alleen het beeldmerk als afbeelding in (als inline SVG), en komen het woordmerk, de 'ribbon' en de applicatienaam terug als tekst. Dit maakt het gebruik van het logo flexibeler en toegankelijker, en bovendien konden we het bepaald gewenst responsive gedrag meegeven.
+
+Vanaf nu kun je op deze manier het logo tonen in je applicatie:
+
+```html
+<dso-logo label="Regels op de kaart" ribbon="beta"></dso-logo>
+```
+
+De applicatienaam geven we mee als `label` op het `dso-logo` component. Dit `label` is optioneel.
+
+Het 'beta' blokje dat in deze fase op het logo staat kan worden ingezet door het logo component het attribuut `ribbon` mee te geven met de gewenste tekst. Zo kunnen we het logo straks ook gebruiken om visueel te communiceren dat we bijvoorbeeld op de pre- of acceptatie omgeving zijn. Ook dit attribuut is optioneel.
+
+### De header verandert ook
```

Comment:
Ik zou ergens benadrukken dat het een breaking change is omdat de API van het Header component anders is. Daar trekken we lekker de aandacht mee ;-)

Reply:
Ik pas de tekst aan!